src/__test__/cart.spec.ts

```diff
@@ -0,0 +1,123 @@
+// import { describe, it } from "mocha";
+// import chai, { expect } from "chai";
```

Comment:
Why is the entire file commented?

Reply:
I needed a separate file to test my endpoint, but it required me to duplicate some code from the product. Instead, I chose to use the same file with the product. But I forgot to delete the cart file. The issue is now resolved.